### PR TITLE
Open modal dialog when using inline place accessibility editing

### PR DESCRIPTION
**Pull request by mutaphysis**
- renamed presetStatus to accessibilityPresetStatus
- removed old url & query handling for the inline button row
- fixed some minor flow errors

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -563,9 +564,9 @@ class Loader extends React.Component<Props, State> {
 
   onSelectWheelchairAccessibility = (value: YesNoLimitedUnknown) => {
     if (this.props.featureId) {
-      this.props.history.push({
-        pathname: `/beta/nodes/${this.props.featureId}/edit-wheelchair-accessibility`,
-        search: `presetStatus=${value}`,
+      this.setState({
+        modalNodeState: 'edit-wheelchair-accessibility',
```
Comment (lennerd) [inline src/App.js]: -1: Should the `modalNodeState` be part of the actual route so we do not have two state machines but one based on routing?

Comment (mutaphysis) [reply src/App.js]: Last discussion status with @opyh & you on Thursday was that we wanted to remove this from the routes. I guess we can bring all of that back, but its unrelated to the fix, the removed lines don't even work.

Comment (lennerd) [reply src/App.js]: Really? Than I changed my mind. Anyway, you're right. This is probably not part of this PR.

Comment (mutaphysis) [reply src/App.js]: I am all in for fewer state machines!

Comment (lennerd) [reply src/App.js]: Should we add a ticket in Trello?

**Pull request merged**